packages/browser/src/eventbuilder.ts

```diff
@@ -71,10 +71,10 @@ export function eventFromPlainObject(
     },
   };
 
-  if (syntheticException) {
+  if (syntheticException && event.exception?.values?.[0]) {
```

Comment:
This check is redundant since we know it's set above. Is there any other option?

Reply:
can we not cast as `Event` above, and just return `event as Event` remove this typecheck?